rdrf/rdrf/forms/registration_forms.py

```diff
@@ -0,0 +1,122 @@
+from operator import attrgetter
+import pycountry
+from django.forms import CharField, ChoiceField, DateField, BooleanField
+from django.forms.widgets import RadioSelect, Select
+from django.utils.translation import gettext as _
+
+from registration.forms import RegistrationForm
+from rdrf.helpers.utils import get_preferred_languages
+
+
+def _tuple(code, name):
+    return code, _(name)
+
+
+def _countries():
+    countries = sorted(pycountry.countries, key=attrgetter('name'))
+    result = [_tuple("-1", "Country")]
```

Comment:
Shouldn't this be "" instead of "-1" so that the required validator kick in?

Reply:
Updated.